openbas-api/src/main/java/io/openbas/injectors/manual/ManualContract.java

```diff
@@ -42,9 +45,16 @@ public ContractConfig getConfig() {
 
   @Override
   public List<Contract> contracts() {
+    ContractElement teams = teamField("teams", "Teams", Multiple);
```

Comment:
We can have a default constructor here since every call have the same properties.
What do you think ?